README.md

```diff
@@ -0,0 +1,63 @@
+# Gramine-RATLS-Wrapper
+
+Gramine-RATLS-Wrapper is a Go wrapper on top of the [Gramine](https://github.com/gramineproject/gramine) [Remote Attestation TLS](https://github.com/gramineproject/gramine/tree/master/tools/sgx/ra-tls) (RATLS) library. Forwards the certificate and SGX quote to the DCAP verification service. It also verifies SGX specific measurement variables that are used to verify the identity of the enclave, namely:
+
+1. `MRENCLAVE`: hex string that verifes attesting enclave.
+2. `MRSIGNER`: hex string that verifies the key used to sign the enclave.
+3. `ISV_PROD_ID`: decimal string that verifes the valid product ID.
+4. `ISV_SVN`: decimal string metadata variable that specifes valid version number.
+
+More info about the core library can be found at the official gramine [docs](https://github.com/gramineproject/gramine).
```

Comment:
Is this the wrong link? -> https://gramine.readthedocs.io/en/stable/attestation.html#mid-level-ra-tls-interface

Reply:
oops. thanks, changed.